### PR TITLE
[MLGROUP-126]Feature/inventory page

### PR DESCRIPTION
**Pull request by SaraLopez03**
El link para esta pagina es: www.prilicores.com/inventario

Se debe de tener las siguientes columnas: Nombre producto, Cantidad, Precio compra, Pecio venta, Utilidad, Acciones

Las columnas: Precio Compra, Precio venta y utilidad deben de mostrarse con el formato de moneda

La columna de acciones debe de mostrar dos botones: Editar y eliminar (Sin funcionalidad)

Se debe de mostrar un boton para agregar producto (Sin funcionalidad)

**All the above requirements are met**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -95,4 +95,26 @@
...
+
+.btn-table{
+  background-color: #dbb82d;
+  border: none;
+  cursor: pointer;
+  font-size: 16px;
+  color: white;
+}
+
+.mr{
```
Comment (DanielMaru) [inline src/index.css]: I think bootstrap already has this class `mr`, [Documentation](https://getbootstrap.com/docs/4.0/utilities/spacing/)

File: src/index.css [hunk 2]
```diff
@@ -95,4 +95,26 @@
...
+  margin-right: 5px;
+}
+
+.btn:first-child:hover{
```
Comment (DanielMaru) [inline src/index.css]: I think we can improve this selector. Creating a new class

File: src/index.css [hunk 3]
```diff
@@ -95,4 +95,26 @@
...
+  background-color:#dbb82d;
+  color: white;
+}
+
+:not(.btn-check)+.btn:hover{
```
Comment (DanielMaru) [inline src/index.css]: I think we can improve this selector. Creating a new class

File: src/index.css (hunk 1 above)
Comment (SaraLopez03) [reply src/index.css]: yes love I know that boostrap has it but the change did not apply to me

File: src/pages/Inventory/InventoryPage.js [hunk 4]
```diff
@@ -1,9 +1,107 @@
...
+
+
+    return(
+        <div>
+            <div className ="mx-4">
+                <h1 className="style-title-page mt-4">Inventario</h1>
+                <button type="button" className="btn btn-primary btn-sm mt-5" onClick={addNewProduct}> <i className="fa-solid fa-plus"></i> Agregar producto </button>
+                <Table columnNames={inventoryColumns} items={items} />
+            </div>
+            {/* <Modal show={newProduct}>
```
Comment (DanielMaru) [inline src/pages/Inventory/InventoryPage.js]: Why do you leave this comment code?

File: src/index.css (hunk 2 above)
Comment (SaraLopez03) [reply src/index.css]: I try to do it with a new class and it does not take the change.

File: src/index.css (hunk 3 above)
Comment (SaraLopez03) [reply src/index.css]: I try to do it with a new class and it does not take the change.

File: src/index.css (hunk 1 above)
Comment (DanielMaru) [reply src/index.css]: Really? Lets check it @SaraLopez03

**Review by DanielMaru — APPROVED** [review]

**Pull request merged**